frontend/src/hooks/useProperties.ts

```diff
@@ -91,6 +94,47 @@ export const useProperties = () => {
     [dispatch, getProperty, getPropertyWfs],
   );
 
+  /**
+   * Make an AJAX request to fetch the specified 'property' from inventory.
+   * @param params PID of the property
+   */
+  const fetchPropertyWithPid = useCallback(
+    async (pid: string): Promise<Api_Property> => {
+      dispatch(logRequest(actionTypes.GET_PARCEL_DETAIL));
+      dispatch(showLoading());
+      // Due to spatial information being stored in BC Albers in the database, we need to make TWO requests here:
+      //   1. to the REST API to fetch property field attributes (e.g. address, etc)
+      //   2. to GeoServer to fetch latitude/longitude in expected web mercator projection (EPSG:4326)
```

Comment:
I'm no expert, but can't leaflet handle projections as well? that way we can save a request.

Reply:
ie something like: L.Projection.Mercator.project(latlng);

Reply:
I'd have to check. I was under the impression that leaflet wouldn't handle the specific projection in our DB (BC Albers) as it is not one of the popular ones. There indeed are client-side projection libraries (turf-js) but we haven't included that in our project (yet). We might look at refactoring this code (and other places where we do the same) via a local client-side re-projection library. 

It would be outside of this PR though

Reply:
sorry - meant https://github.com/proj4js/proj4js
turf-js is more for shape manipulation and measuring client side